storybook/stories/LinePlot.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import ndarray from 'ndarray';
+import { useArgs } from '@storybook/preview-api';
+import type { Meta, StoryObj } from '@storybook/react';
+import {
+  BatonProps,
+  DAxesParameters,
+  DLineData,
+  SelectionBase,
+  LinePlot,
+} from '@davidia/component';
+
+const meta: Meta<typeof LinePlot> = {
+  title: 'Plots/Line',
+  component: LinePlot,
+  tags: ['autodocs'],
+};
+
+export default meta;
+
+const batonProps = {
```

Comment:
I think this should be unnecessary for any static plot.

Reply:
The way it's currently set up with PlotSelectionProps is that every plot needs batonprops even if they're not used.

Reply:
Can you change it so that an undefined `batonProps` is equivalent to `hasBaton = true`?